benchmarking/nursery/benchmark_neuralband/benchmark_main.py

```diff
@@ -0,0 +1,214 @@
+            configuration_space=bb.configuration_space,
+            hyperparameters=bb.hyperparameters,
+            objectives_evaluations=bb.objectives_evaluations[
+                ..., metric_index : metric_index + 1
+            ],
+            objectives_names=[bb.objectives_names[metric_index]],
+        )
+        for task, bb in task_to_evaluations.items()
+        if task != test_task and (datasets is None or task in datasets)
+    }
+
+    if n_evals is not None:
+        # subsample n_evals / n_tasks of observations on each tasks
+        def subsample(
+            transfer_evaluations: TransferLearningTaskEvaluations, n: int
+        ) -> TransferLearningTaskEvaluations:
+            random_indices = np.random.permutation(
+                len(transfer_evaluations.hyperparameters)
+            )[:n]
+            return TransferLearningTaskEvaluations(
+                configuration_space=transfer_evaluations.configuration_space,
+                hyperparameters=transfer_evaluations.hyperparameters.loc[
+                    random_indices
+                ].reset_index(drop=True),
+                objectives_evaluations=transfer_evaluations.objectives_evaluations[
+                    random_indices
+                ],
+                objectives_names=transfer_evaluations.objectives_names,
+            )
+
+        n = n_evals // len(transfer_learning_evaluations)
+        transfer_learning_evaluations = {
+            task: subsample(transfer_evaluations, n)
+            for task, transfer_evaluations in transfer_learning_evaluations.items()
+        }
+
+    return transfer_learning_evaluations
+
+
+if __name__ == "__main__":
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--experiment_tag", type=str, required=False, default=generate_slug(2)
+    )
+    parser.add_argument(
+        "--num_seeds",
+        type=int,
+        required=False,
+        default=1,
+        help="number of seeds to run",
+    )
+    parser.add_argument(
+        "--run_all_seed",
+        type=int,
+        default=1,
+        help="if 1 run only `seed=num_seeds`, otherwise runs all the seeds [0, `num_seeds`-1]",
+    )
+    parser.add_argument(
+        "--method", type=str, required=False, help="a method to run from baselines.py"
+    )
+    parser.add_argument(
+        "--benchmark",
+        type=str,
+        required=False,
+        help="a benchmark to run from benchmark_definitions.py",
+    )
+    args, _ = parser.parse_known_args()
+    experiment_tag = args.experiment_tag
+
+    if args.run_all_seed == 1:
+        seeds = list(range(args.num_seeds))
+    else:
+        seeds = [args.num_seeds]
+
+    method_names = [args.method] if args.method is not None else list(methods.keys())
+
+    
+    benchmark_names = (
+        [args.benchmark]
+        if args.benchmark is not None
+        else list(benchmark_definitions.keys())
+    )
+
+    #benchmark_names = ['fcnet-protein']
+
+    print(benchmark_names)
```

Comment:
Seems like an old debug message, could you remove it?

Reply:
Sure, done.